File: DSA/String/3110_Score_of_a_String/solution.cpp
// 3110. Score of a String
// Difficulty : Easy
// Tags       : string
// Solved on  : 2025-10-20
// LeetCode   : https://leetcode.com/problems/score-of-a-string/

class Solution {
public:
    int scoreOfString(string s) {
        int score = 0;
        for(int i = 0; i<s.length() -1;i++)
        {
            score+= abs(s[i]-s[i+1]);
        }
        return score;
    }
};

File: DSA/String/3110_Score_of_a_String/solution.py
# 3110. Score of a String
# Difficulty : Easy
# Tags       : string
# Solved on  : 2025-10-20
# LeetCode   : https://leetcode.com/problems/score-of-a-string/

class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0
        for i in range(0,len(s)-1):
            score = score + abs(ord(s[i])-ord(s[i+1]))
        return score
        